### ISSUE TITLE
kfctl apply failed -- no default network

### ISSUE THREAD

**Issue by LukasBlank** [kubeflow/kubeflow#3616]
I am following the deploy using the cli guide (https://www.kubeflow.org/docs/gke/deploy/deploy-cli/) running:
<pre><code>kfctl init ${KFAPP} --platform gcp --project ${PROJECT} --use_basic_auth -V</code><br>
<code>kfctl generate all -V --zone ${ZONE}</code>
<br>
<code>kfctl apply all -V</code>
</pre>
KFAPP, PROJECT and ZONE are set correctly and before running these commands i did
<pre><code>gcloud auth application-default login</code></pre>
which seemed to work. 
kfctl is installed and i do move to the created directory after running init.
I do get the Error:
<pre><code>Creating kfapp error: &{Code:RESOURCE_ERROR Location:/deployments/kfapp/resources/kfapp Message:{"ResourceType":"gcp-types/container-v1beta1:projects.locations.clusters","ResourceErrorCode":"400","ResourceErrorMessage":{"code":400,"message":"Project \"my_project\" has no network named \"default\".","status":"INVALID_ARGUMENT","statusMessage":"Bad Request","requestPath":"https://container.googleapis.com/v1beta1/projects/my_project/locations/europe-west4-a/clusters","httpMethod":"POST"}} ForceSendFields:[] NullFields:[]}  filename="gcp/gcp.go:278"
Error: couldn't apply KfApp:  (kubeflow.error): Code 500 with message: coordinator Apply failed for gcp:  (kubeflow.error): Code 400 with message: gcp apply could not update deployment manager Error could not update cluster-kubeflow.yaml: Creating kfapp error(400): BAD REQUEST
Usage:
  kfctl apply [all(=default)|k8s|platform] [flags]

Flags:
  -h, --help      help for apply
  -V, --verbose   verbose output default is false

couldn't apply KfApp:  (kubeflow.error): Code 500 with message: coordinator Apply failed for gcp:  (kubeflow.error): Code 400 with message: gcp apply could not update deployment manager Error could not update cluster-kubeflow.yaml: Creating kfapp error(400): BAD REQUEST
</code></pre>
He was able to create the kfapp-storage though.

**Comment by LukasBlank** [comment]
Dont now if that helps but i already tried to set GOOGLE_APPLICATION_CREDENTIALS="" before executing these steps and checked that it was not set again. Didnt work neither.

**Comment by LukasBlank** [comment]
Tried to edit the created config files in .cache/master/deployment/gke/deployment-manager-configs before generating it, did not work. Furthermore I followed the steps from https://www.kubeflow.org/docs/gke/troubleshooting-gke/#legacy-networks-are-not-supported, but that did not work. The errror remained the same and the changes seemed to have no effect on the result i am getting. Does someone know how i can tell kubeflow to use an other network than the default one?
